### DISCUSSION TITLE
Steam ?

### DISCUSSION THREAD

**Discussion by JamesDAdams** [vogler/free-games-claimer#86] [Ideas]
Hi, 

it's possible to add Steam ?

Thanks for your soft <3

**Comment by vogler** [comment]
Pretty much everything you can do in the browser is possible. Someone just has to implement it 😄
What set of games do you have in mind? I guess 'free to play' are not of interest.

This seems like a reasonable query:
https://store.steampowered.com/search/?sort_by=Reviews_DESC&as-hide=negative&maxprice=free&category1=998%2C21&specials=1&hidef2p=1&ndl=1

**Comment by ladiko** [comment]
The URL param `sort_by=Reviews_DESC` leads to one result without enough reviews being hidden. Not sure if that's intended? And usually yes: quality over quantity, but value-for-money is always quite okay if its free. `maxprice=free&specials=1&hidef2p=1` seems to be a good minimal starting point. Not sure yet what `as-hide=negative` and `ndl=1` do.

**Reply by vogler** [reply]
Confirm, should use https://store.steampowered.com/search/?maxprice=free&specials=1&hidef2p=1
`as-hide=negative` is 'Hide negative rating items'
`ndl=1` doesn't seem to make a difference but is added after you modify the search options.

**Comment by alzk** [comment]
It is easier here https://steamdb.info/upcoming/free/

where in the Type tag if it has keep then I would claim it, as long as it is not a dlc.

**Reply by vogler** [reply]
That may be a problem:
![image](https://user-images.githubusercontent.com/493741/224262712-7ccecb84-0e81-4824-9403-fd25da4de8cd.png)

The 'keep' results match the updated search URL above:
![image](https://user-images.githubusercontent.com/493741/224263230-cbd58658-cac3-444e-8eda-7d4933415879.png)
![image](https://user-images.githubusercontent.com/493741/224265034-a4d8f46f-a5e5-4745-8772-2b08eb116d08.png)

**Comment by zephiowl** [comment]
Any status on this? It's a feature I'd love to see and its been a few months since the initial idea

**Reply by vogler** [reply]
Busy with work. Also wanted to fix other stuff first. You can try to make a PR.

**Reply by zephiowl** [reply]
Not really a JS dev but I could always try it out and see how it goes.
